templates/show.html

```diff
@@ -0,0 +1,48 @@
+{% extends 'base.html' %}
+{% block head %}
+  {% load static %}
```

Comment:
Same again.

I prefer to keep {% load %} tags immediately below the {% extend %} so I know where to find them.

(In base templates I put them immediately following the <!DOCTYPE html> so they don't create a blank line :)